ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -257,6 +263,42 @@ export default class TestDataModal extends React.Component {
     this.closeModal();
   };
 
+  selectableTestClassName = test => {
+    if (this.hasDifferentUnit(test)) {
```

Comment:
Can we use alternative `if` syntax?
`return this.hasDifferentUnit(test) ? 'bg-warning' : '';`

Reply:
Yep.